packages/datadog-plugin-pg/test/index.spec.js

```diff
@@ -404,8 +403,8 @@ describe('Plugin', () => {
             if (strBuf.includes('traceparent=\'')) {
               strBuf = strBuf.split('-')
               seenTraceParent = true
-              seenTraceId = strBuf[1]
-              seenSpanId = strBuf[2]
```

Comment:
Were these old indexes incorrect?

Reply:
It was correct before, by removing the service name in the plugin it now defaults to the tracer service name which contains a hyphen and so the array created by the split is now thrown off by 1.